packages/trader/src/Modules/CFD/Components/cfd-poa.jsx

```diff
@@ -60,7 +60,7 @@ const CFDPOA = ({ onSave, onCancel, index, onSubmit, refreshNotifications, ...pr
             address_state: [v => !!v && !v.match(/^\s*$/), v => !v || validLength(v, { min: 1, max: 35 })],
             address_postcode: [
                 v => !!v && !v.match(/^\s*$/),
-                v => validLength(v, { min: 1, max: 20 }),
+                v => !!v && validLength(v, { min: 1, max: 20 }),
```

Comment:
This change won't be needed after we set an empty string as default to value param in `validLength` function.